Orange/widgets/visualize/owheatmap.py

```diff
@@ -320,9 +320,9 @@ def __new__(cls, title, indices, sortindices, cluster=None,
     @property
     def is_empty(self):
         if isinstance(self.indices, slice):
-            return (self.indices.stop - self.indices.start) == 0
+            return (self.indices.stop - self.indices.start) < 2
         else:
-            return len(self.indices) == 0
+            return len(self.indices) < 2
```

Comment:
The name of the property becomes a bit awkward (=wrong) after this fix, since it does not check if the group is empty anymore, but if it contains 0 or 1 items.
However, it is only used in one place and apart from the exotic definition of "empty" it works as it should.
I am fine merging this as is.
Possible fixes (if you feel it is worth it) would be for example
- inverting the property to `has_multiple_items` (can't think of a better name) and dropping the `not` where it is called
- remove the property and implement `__len__` instead and check that it is `< 2` in the `cluster_rows` method (only use of the property)